aws/data_source_aws_security_group.go

```diff
@@ -38,6 +38,12 @@ func dataSourceAwsSecurityGroup() *schema.Resource {
 			},
 
 			"tags": tagsSchemaComputed(),
+
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Optional is used when the user can pass a value for this attribute. Since we are only retrieving the description (and we don't need it to retrieve the Security Group info), this is not needed here and can be removed